pyccel/parser/semantic.py

```diff
@@ -1132,6 +1136,73 @@ def _handle_function(self, expr, func, args, *, is_method = False, use_build_fun
                             func, func.is_elemental)
 
             return new_expr
+    def _handle_kernel(self, expr, func, args, **settings):
+        """
+        Create the node representing the kernel function call.
+
+        Create a FunctionCall or an instance of a PyccelInternalFunction
+        from the function information and arguments.
+
+        Parameters
+        ----------
+        expr : IndexedFunctionCall
+               Node has all the information about the function call
+
+        func : FunctionDef instance, Interface instance or PyccelInternalFunction type
+               The function being called.
+
+        args : tuple
+               The arguments passed to the function.
+
+        **settings : dict
+            The settings passed to _visit functions.
```

Comment:
The settings argument was removed a while ago

Reply:
fixed , https://github.com/pyccel/pyccel-cuda/commit/730e69b2b3a494dc7575c10f0208bab38e68a61f

Reply:
Not fixed, you have changed the docstring but not the arguments or the function call

Reply:
fixed 3e7d535